lib/utils/background_push.dart

```diff
@@ -545,6 +542,14 @@ class BackgroundPush {
     }
   }
 
+  String? get roomId {
+    final lastMatch = router?.routerDelegate.currentConfiguration.last;
```

Comment:
you should create an extension for GoRouter then we can reuse
- one func return pathParameters
- one func return currentContext

Reply:
IMO: IMO: Workaround is ugly. On another screen we have `context` and can use it. I think don't need it. Because in this case, It's class and doesn't have `BuildContext` so needs a global `context` in here. Just only herreee

Reply:
> IMO: 
- Workaround is ugly. On another screen we have `context` and can use it. I think don't need it. Because in this case, It's class and doesn't have `BuildContext` so needs a global `context` in here. Just only herreee.
- You cannot arbitrarily use `GoRouter` everywhere